pkg/kubelet/cm/types.go

```diff
@@ -49,3 +49,16 @@ type CgroupManager interface {
 	// Update cgroup configuration.
 	Update(*CgroupConfig) error
 }
+
+// QOSConfig defines how the qos cgroup hierarchy is organized
+type QOSConfig struct {
+	// RootContainerName is the root for the qos hierarchy
+	RootContainerName string
+}
+
+// QOSContainersInfo hold the names of containers per qos
+type QOSContainersInfo struct {
```

Comment:
I thought that this would be good have. Once the top level QOS containers are created they won't change. So we can store there names in the cm and it can be used for getting the absolute names of the pod containers.